_layouts/location.html

```diff
@@ -6,6 +6,330 @@
 {% assign location_happenings = site.promos | where: "target_audience", page.name | sort: "date" | reverse %}
 {% include header.html %}
 
+
+<!-- Christmas Banner -->
+<div class="jumbotron inline-video jumbotron-md flush-bottom" style="background-image: url('//crds-cms-uploads.imgix.net/content/images/crossroads-christmas-drummer-bg-img.jpg');">
+  <div class="inline-video-player"></div>
+  <div class="jumbotron-content">
+    <div class="container-fluid">
+      <div class="row soft-sides">
+        <div class="col-sm-6 col-sm-offset-3">
+          <img data-src="https://crds-cms-uploads.imgix.net/content/images/crossroads-christmas-logo.png" class="img-responsive" data-optimize-img />
+        </div>
+      </div>
+      <div class="row push-ends soft-ends">
+        <div class="col-sm-12 push-ends soft-ends">
+          <a href="#" class="inline-video-trigger push-ends soft-ends" data-video-id="LIdCR2rywl4"></a>
+        </div>
+      </div>
+      <div class="row soft-sides push-top soft-top">
+        <div class="col-sm-8 col-sm-offset-2 push-half-top">
+          <p>Feel the wonder of Christmas in a completely new way. This hour-long service includes fully immersive teaching and music that reminds you why Christmas is the ultimate season of hope. Watch the trailer above to get a sneak peek.</p>
+          <span class="font-size-small">(No childcare, but your kids will love it.)</span><br>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+{% if page.title == "Andover" %}
+  <section style="background-color:#970111;">
+    <div class="container soft">
+      <div class="row push-ends">
+        <div class="col-sm-4 soft-sides">
+          <h2 class="component-header text-white flush-top">Andover Service Times</h2>
```

Comment:
We can simply this `if` with a few tricks. One is this: 

```suggestion
          <h2 class="component-header text-white flush-top">{{ page.title }} Service Times</h2>
```